mpas_analysis/sea_ice/timeseries.py

```diff
@@ -14,6 +11,9 @@
 from ..shared.timekeeping.utility import stringToDatetime, \
     clampToNumpyDatetime64
 
+from ..shared.MpasAnalysisXarray.MpasAnalysisXarray import openMultifileDataSet
+from ..shared.MpasXarray.MpasXarray import subsetVariables
```

Comment:
You are right @xylar, the organization here is important and need further consideration.  I'm thinking we probably do need a design doc

 1. highlighting directory structure
 2. highlighting public API

I don't think this needs to be a "full" design doc, but more of a simple listing of how this all should work.